code/game/objects/items/weapons/tech_disks.dm

```diff
@@ -1,18 +1,34 @@
 /obj/item/disk
-	icon = 'icons/obj/cloning.dmi'
-	icon_state = "datadisk2"
-	item_state = "card-id"
-	w_class = ITEM_SIZE_SMALL
-	matter = list(/decl/material/solid/plastic = 30, /decl/material/solid/metal/steel = 30, /decl/material/solid/glass = 10)
+	name = "data disk"
+	icon = 'icons/obj/items/device/diskette.dmi'
+	icon_state = ICON_STATE_WORLD
+	w_class = ITEM_SIZE_TINY
+	material = /decl/material/solid/plastic
+	matter = list(/decl/material/solid/metal/steel = MATTER_AMOUNT_TRACE)
+	var/label = "label_blank"
+	
+/obj/item/disk/on_update_icon()
+	. = ..()
+	cut_overlays()
+	var/list/details = list()
+	details += mutable_appearance(icon, "slider", flags = RESET_COLOR)
```

Comment:
Why use MAs for this?

Reply:
whatever proc I rememberd with reset colors flag settable in constructor, not much beyond that.